### DISCUSSION TITLE
Cannot see Variable in output file

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#18168] [Q&A General] [answered]
After simulation was done, I opened the result(*.e) through Paraview.

<img width="264" alt="Screen Shot 2021-06-25 at 3 44 31 AM" src="https://user-images.githubusercontent.com/46009457/123389398-d5c90180-d567-11eb-8d0b-7ee4ae7c57a5.png">

However, as shown at above, I cannot see one of variables (phi_Ni) I assigned for the simulation.

<img width="470" alt="Screen Shot 2021-06-25 at 3 43 48 AM" src="https://user-images.githubusercontent.com/46009457/123389570-03ae4600-d568-11eb-81cd-f393afd87f69.png">

During the simulation, I saw messaged like above.

"Outlier Variable Residual Norms".

Is this related to not seeing the variable in the output file?

**Comment by GiudGiud** [comment, marked as answer]
Hello

the outlier variable residual norm is not related to not seeing the variable. 
this is very odd. Unless you have hide = ‘phi_Ni’ in the output block, I do not understand why it s happening. 

Could you please delete the exodus file and try generating it again? You may be opening an old one or something?

guillaume

**Reply by hokonkim** [reply]
I read my input file thoroughly.

I just got a point.

'phi_Ni' existed at [Variables] and [Postprocessors].

After I changed the name in [Postprocessors], then now I can see the variable in output file.
